memory_statistics/action.yml

```diff
@@ -30,3 +34,15 @@ runs:
           --config "${{ inputs.config }}"
           --output "${{ inputs.output }}"
       shell: bash
+    - name: Compare table
+      run: |
+          if [ "x${{ inputs.check_against }}" != "x" ] ; then
```

Comment:
Does this not work to check if the input variable is empty?
`[ -z "${{ inputs.check_against }}" ]`